Compiler/Util/Flags.mo

```diff
@@ -509,6 +509,9 @@ constant DebugFlag PARTITION_INITIALIZATION = DEBUG_FLAG(167, "partitionInitiali
   Util.gettext("This flag controls if partitioning is applied to the initialization system."));
 constant DebugFlag EVAL_PARAM_DUMP = DEBUG_FLAG(168, "evalParameterDump", false,
   Util.gettext("Dumps information for evaluating parameters."));
+constant DebugFlag NF_UNITCHECK = DEBUG_FLAG(169, "frontEndUnitCheck", false,
+  Util.gettext("Checks the consistency of units in equation."));
+
```

Comment:
There is also the old flag `unitChecking` which is used to activate the old front end implementation. If that’s not used anymore, we could remove that implementation and reuse the old config flag.